code/game/machinery/atmoalter/area_atmos_computer.dm

```diff
@@ -23,159 +19,52 @@
 	return src.attack_hand(user)
 
 /obj/machinery/computer/area_atmos/attack_hand(var/mob/user as mob)
-	if(..(user))
-		return
-	src.add_fingerprint(usr)
-	var/dat = {"
-	<html>
-		<head>
-			<style type="text/css">
-				a.green:link
-				{
-					color:#00CC00;
-				}
-				a.green:visited
-				{
-					color:#00CC00;
-				}
-				a.green:hover
-				{
-					color:#00CC00;
-				}
-				a.green:active
-				{
-					color:#00CC00;
-				}
-				a.red:link
-				{
-					color:#FF0000;
-				}
-				a.red:visited
-				{
-					color:#FF0000;
-				}
-				a.red:hover
-				{
-					color:#FF0000;
-				}
-				a.red:active
-				{
-					color:#FF0000;
-				}
-			</style>
-		</head>
-		<body>
-			<center><h1>Area Air Control</h1></center>
-			<font color="red">[status]</font><br>
-			<a href="?src=\ref[src];scan=1">Scan</a>
-			<table border="1" width="90%">"}
+	ui_interact(user)
+	return
+
+/obj/machinery/computer/area_atmos/ui_interact(mob/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "AreaAtmos", ui_x=500, ui_y=300)
```

Comment:
`ui_x` and `ui_y` are deprecated, the size should be specified inside the UI itself, in the `Window` component in particular, as per https://github.com/Aurorastation/Aurora.3/blob/master/tgui/docs/component-reference.md#window

You already did this, possibly forgot to remove the two vars from here?